npm/private/npm_translate_lock_generate.bzl

```diff
@@ -121,6 +121,7 @@ js_binary(name = "sync", entry_point = "noop.js")
                 transitive_deps[dep_store_target] = ",".join(transitive_deps[dep_store_target])
             fp_links[dep_key] = {
                 "package": name,
+                "version": version,
```

Comment:
This causes a `.get("version", "0.0.0")` to now use the raw `file:` version instead of dropping it. Then the change in `utils.bzl` is the other spot that was previously dropping `file:` versions.